x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx

```diff
@@ -55,7 +55,7 @@ export function StickyTransactionProperties({
     },
     {
       label: '% of trace',
-      val: asPercent(duration, totalDuration),
+      val: asPercent(duration, totalDuration, 'N/A'),
```

Comment:
Code review is a place where destructuring is nice IMO, e.g.
```
asPercent({ numerator: duration, denominator: totalDuration, fallback: 'N/A' }) 
```
mostly for the fallback value to have some context ... but let's not change that right now at the release deadline :)

Reply:
I can see how it makes it more readable here.